src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -101,6 +102,11 @@ public class GenerateScriptCommand : CommandBase<GenerateScriptCommandArgs, Gene
             Description = "If using AWS, the name of the S3 bucket to upload the BBS archive to."
         };
 
+        public Option<string> AwsRegion { get; } = new("--aws-region")
+        {
+            Description = "If using AWS, the AWS region. If not provided, it will be read from AWS_REGION environment variable. Defaults to us-east-1."
```

Comment:
```suggestion
            Description = "If using AWS, the AWS region. If not provided, it will be read from AWS_REGION environment variable. Defaults to us-east-1 if neither the argument nor the environment variable is set. In a future release, you will be required to set an AWS region if using AWS S3 as your blob storage provider."
```

Reply:
Addressed in 98de164b93e6c5ad0a9f24c52673cd80b4f9a7fc